modules/ppcp-button/src/Endpoint/CreateCrossBrowserOrderEndpoint.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * The endpoint to create WooCommerce order for cross-browser AppSwitch flows.
+ *
+ * @package WooCommerce\PayPalCommerce\Button\Endpoint
+ */
+
+declare(strict_types=1);
+
+namespace WooCommerce\PayPalCommerce\Button\Endpoint;
+
+use Exception;
+use WooCommerce\PayPalCommerce\ApiClient\Endpoint\OrderEndpoint;
+use WooCommerce\PayPalCommerce\Button\Helper\WooCommerceOrderCreator;
+use WooCommerce\PayPalCommerce\Button\Session\CartDataTransientStorage;
+use WooCommerce\PayPalCommerce\WcGateway\Gateway\PayPalGateway;
+
+class CreateCrossBrowserOrderEndpoint implements EndpointInterface {
+
+	const ENDPOINT = 'ppc-create-cross-browser-order';
```

Comment:
Please add visibility.
```suggestion
	public const ENDPOINT = 'ppc-create-cross-browser-order';
```